db/migrate/20240722064452_create_likes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLikes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+      t.string :liked_what
+      t.integer :liked_id
+      t.integer :user_id
```

Comment:
these columns should be reference columns. Add foreign key constraints on these columns. read how we implement this concept in Rails.